cmd/agentctl/client/client.go

```diff
@@ -150,7 +151,24 @@ func (c *Client) GRPCConn() (*grpc.ClientConn, error) {
 // HTTPClient returns configured HTTP client.
 func (c *Client) HTTPClient() *http.Client {
 	if c.httpClient == nil {
-		tr := http.DefaultTransport.(*http.Transport).Clone()
+		// tr is copy-pasted definition of DefaultTransport from the net/http package
```

Comment:
You can actually use build tag for this. Check out this file from std package `sort` to see how.

https://github.com/golang/go/blob/0a338f75d4/src/sort/slice_go113.go

Reply:
woooow, didn't get it at first. So this is how you support different versions of Go (TIL). Good to know, I'll consider this